chainer/_backend.py

```diff
@@ -105,3 +105,16 @@ def use(self):
         """Makes the device current in the current thread.
          """
         pass
+
+    def is_array_compatible(self, array):
```

Comment:
`is_array_compatible` needs its own tests.
Could you separate a PR?

Reply:
I see. Moved to #7487.